.github/workflows/main-docs-deploy.yml

```diff
@@ -32,10 +32,17 @@ jobs:
         uses: actions/checkout@v4
       - name: Setup Pages
         uses: actions/configure-pages@v5
-      - uses: ammaraskar/sphinx-action@dev
+      - name: Install system dependencies
+        uses: awalsh128/cache-apt-pkgs-action@latest
```

Comment:
What is the difference between using this thirdy-party action and the official actions/cache from github?